src/main/kotlin/three/consulting/epoc/config/SecurityConfig.kt

```diff
@@ -3,28 +3,48 @@ package three.consulting.epoc.config
 import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
 import org.springframework.context.annotation.Profile
+import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity
 import org.springframework.security.config.annotation.web.builders.HttpSecurity
-import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity
+import org.springframework.security.oauth2.server.resource.authentication.JwtAuthenticationConverter
+import org.springframework.security.oauth2.server.resource.authentication.JwtGrantedAuthoritiesConverter
 import org.springframework.security.web.SecurityFilterChain
+import org.springframework.stereotype.Component
 import org.springframework.web.servlet.config.annotation.CorsRegistry
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer
 
 @Profile("default")
-@EnableWebSecurity
+@Component
+@EnableGlobalMethodSecurity(
+    prePostEnabled = true
+)
 class SecurityConfig() {
 
+    val authoritiesClaimName = "role"
```

Comment:
In kotlin it's very common to wrap constant values inside a companion object for clarity and define them as private, and write in caps snake case constants if needed
```kotlin
private companion object {
    const val AUTHORITIES_CLAIM_name = "role"
}
```